RELEASE.md

```diff
@@ -0,0 +1,32 @@
+# Release process
+
+- [Release reference #1](https://github.com/apiaryio/dredd/pull/1626)
+- [Release reference #2](https://github.com/apiaryio/dredd/pull/1862)
+
+## 1. Create a release pull request
+
+```bash
+$ git checkout -b <user>/dredd-14.0.0
+$ npx lerna version --no-push --no-git-tag-version 
+$ git add .
+$ git push -u origin <user>/dredd-14.0.0
+```
+
+## 2. Post-merge process
+
+**Once the release pull request is approved and merged**, proceed using the following instructions:
+
+```bash
+$ git checkout master
+$ git pull
```

Comment:
no `npm install` or `yarn install` post-checkout?

Reply:
What I mean is that unless you don't bundle any packages, it'll work. But AFAIK Dredd or Dredd-Transactions have bundled one or two packages? So the install of packages might be required. But anyway. Good enough to merge this Docs. It's already a good improvement to "nothing" 👍

Reply:
That's a good point. From what I see `npx lerna publish` calls `npm pack` on each package, and we have pre- and postpack scripts specified where needed:

https://github.com/apiaryio/dredd/blob/11e821a482745e89315e0832ed6c52c751755e73/packages/dredd-transactions/package.json#L10-L11

You'd have to have dependencies installed, that's true. If they are missing you'd get an exception. The risk is to bundle a mismatched version of the package. 

Do you suggest to add `yarn install` as an error-safe step to ensure properly bundled dependencies?

Reply:
@artem-zakharchenko adding `yarn install` as a pre-requisite for the release? Yep, exactly.